maintainers/maintainer-list.nix

```diff
@@ -7403,6 +7403,12 @@
     githubId = 13752145;
     name = "Richard Lupton";
   };
+  rmcgibbo = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add rmcgibbo``.